scripts/gulp/serve-package.js

```diff
@@ -43,7 +43,7 @@ function servePackage(port, hostname) {
 
   createServer(app).listen(port, () => {
     const scheme = useSsl ? 'https' : 'http';
-    log(`Package served at ${scheme}://${hostname}:${port}/hypothesis`);
+    log(`Package served at ${scheme}://localhost:${port}/hypothesis`);
```

Comment:
You explained to me why this `localhost` is hard-coded on a call; would a comment here be useful?

Reply:
Mmm ... I'm not sure what I can usefully write here. I'll see whether any issues come up when people start testing this.